docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -114,8 +114,8 @@ Below is a list of separate examples for different frameworks and model conversi
           .. code-block:: py
              :force:
 
-             from openvino.tools.mo import convert_model
-             ov_model = convert_model("BERT", input_shape=[[2,30],[2,30],[2,30]])
+             import openvino as ov
+             ov_model = ov.convert_model("BERT", input_shape=[[2,30],[2,30],[2,30]])
```

Comment:
Please revert changes in all convert_model() examples. 
This will not work.

Same comment for other convert_model() examples.

Reply:
I have reverted them in the https://github.com/openvinotoolkit/openvino/pull/19084/commits/24d1321aa23402e051f7925957e6eedaacd7d9ff commit.